easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb

```diff
@@ -0,0 +1,137 @@
+name = 'Python'
+version = '2.7.11'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.17.0'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.2'),
```

Comment:
6.3 doesn't work for me when using ipython. See here 

https://github.com/hpcugent/easybuild-easyconfigs/pull/2457#issuecomment-220324246
https://github.com/hpcugent/easybuild-easyconfigs/pull/2457#issuecomment-220549806

In any case if you want librealine 6.3 I will change it